README.md

```diff
@@ -37,7 +37,7 @@ We're really appreciate your help 😊
 
 Currently, CDNJS is the top 2([ref](https://w3techs.com/technologies/overview/content_delivery/all)) web front-end CDN service with great performance, we fully support [https](https://en.wikipedia.org/wiki/HTTPS), [SPDY](https://en.wikipedia.org/wiki/SPDY), [http/2.0](https://http2.github.io/) and [SRI](https://www.w3.org/TR/SRI/), which will **boost** and **secure** your website with zero configuration. *(note that you'll still need to take care of the server side and application layer of security issues, we just make it better, but can't help too much if you implment a bad practice)*
 
-## latest version url support
+## Latest version url support
```

Comment:
This latest has its own meaning, so just leave it as what it used to be as it would be the url, that's why we are not using "the latest" here.